src/main/java/io/pravega/connectors/flink/dynamic/table/FlinkPravegaDynamicTableSink.java

```diff
@@ -1,11 +1,11 @@
 /**
  * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
- *
+ * <p>
```

Comment:
Is there a reason you added `<p>` here? It seems we don't have this tag in other files.

Reply:
That's strange. I didn't change it during developing